scripts/validate_docstrings.py

```diff
@@ -186,12 +185,11 @@ def signature_parameters(self):
             # accessor classes have a signature, but don't want to show this
             return tuple()
         try:
-            signature = inspect.signature(self.method_obj)
+            params = self.method_obj.__code__.co_varnames
```

Comment:
unfortunately, this does not give the correct result (it seems to give some extra names). Eg 

```
In [156]: pd.DataFrame.apply.__code__.co_varnames
Out[156]: 
('self',
 'func',
 'axis',
 'broadcast',
 'raw',
 'reduce',
 'result_type',
 'args',
 'kwds',
 'frame_apply',
 'op')

```

the last two should not be there